stylesheets/avatar/avatar.css

```diff
@@ -0,0 +1,185 @@
+@import url(../main.css);
+.component-type-flex-container
+{
+    display: flex;
+    flex-direction: column;
+    max-width: 80vw;
+    max-height: 100%;
+    border-radius: 2rem;
+    padding: 1rem 3rem;
+    /* border: 2px solid var(--dark-stone); */
+    cursor: pointer;
+    flex: 1;
+    gap:2rem;
+}
+/* .section-second-sub-heading
+{
+    font-size: 1.6rem;
+    color: var(--light-coco);
+} */
+.image-avatar, 
+.image-avatars-with-outline,
+.text-avatar-container
+{
+    display:flex;
+    flex-wrap: wrap;
+    max-width:80vw;
+    padding: 0.6rem;
+    gap:2.5rem;
+    align-items: center;
+    justify-content: space-between;
+    flex:1;
+}
+.brdr-purple
+{
+   border: 4px solid var(--dark-purple);
+}
+.brdr-blue
+{
+    border: 4px solid var(--light-blue);
+}
+.brdr-green
+{
+    border: 4px solid var(--light-green);
+}
+
+.avatar
+{
+    border-radius:50%;
+    object-fit: cover;
+}
+.avatar-xl ,
+.text-avatar-xl
+{
+    height:13rem;
+    width: 13rem;
+}
+.avatar-lg,
+.text-avatar-lg
+{
+    height:10rem;
+    width:10rem;
+}
+
+.avatar-md,
+.text-avatar-md
+{
+    height:7rem;
+    width:7rem;
+}
+.avatar-sm,
+.text-avatar-sm
+{
+    height:5rem;
+    width:5rem;
+}
+.avatar-xsm,
+.text-avatar-xsm
+{
+    height:3rem;
+    width:3rem;
+}
+.text-avatar-xl,
+.text-avatar-sm ,
+.text-avatar-lg,
+.text-avatar-md,
+.text-avatar-xsm
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color:var(--medium-teal);
+    color: var(--white-color);
+}
+.text-avatar-xl
+{
+    font-size: 5rem;
+}
+.text-avatar-lg
+{
+    font-size: 3rem;
+}
+.text-avatar-md
+{
+    font-size: 2rem;
+}
+.text-avatar-sm
+{
+    font-size: 1rem;
+}
+.text-avatar-xsm
+{
+    font-size: 1rem;
+}
+.code-snippet
+{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    display: none;
+}
+.code-snippet img
+{   
+    /* display:none; */
+    max-height:80vh;
+    max-width:60vw;
+    object-fit: cover;
+}
+/* media query */
+@media only screen and (min-width:500px )and(max-width:1045px) {
```

Comment:
The code looks nice. Just one suggestion, it is a good practice to add media queries at the end.

Reply:
At the end of project?